### DISCUSSION TITLE
Best way to handle auth user timezones?

### DISCUSSION THREAD

**Discussion by jdion84** [filamentphp/filament#10421] [Help]
Let's say my `User` model has a `timezone` attribute which is stored in the database.

I'd like filament to use this for all dates displayed in the system.

Is the best way to achieve this just to add the `->timezone()` method to all of my date pickers & columns via the `configureUsing` method?

**Comment by kishan-pinetco** [comment]
@jdion84 Do you've idea how to display date-time based on the user's local timezone.
